code/datums/station_traits/negative_traits.dm

```diff
@@ -282,7 +282,7 @@
 	report_message = "A radioactive stormfront is passing through your station's system. Expect an increased likelihood of radiation storms passing over your station, as well the potential for multiple radiation storms to occur during your shift."
 	trait_type = STATION_TRAIT_NEGATIVE
 	trait_flags = NONE
-	weight = 0 //SKYRAT EDIT CHANGE - ORIGINAL: weight = 0
+	weight = 1 //BUBBERS EDIT CHANGE - ORIGINAL: weight = 0
```

Comment:
No.

Reply:
Gotcha! Thought it'd be good to experiment but that's not suitable for 1.0